pkg/manifests/manifests_test.go

```diff
@@ -2805,7 +2805,7 @@ metricsServer:
 	}
 	metricsClientSecret := &v1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "metrics-client-cert",
+			Name:      "metrics-server-client-cert",
```

Comment:
the secret name has an `s` at the end `metrics-server-client-certs`, I don't see how the tests were passing...